source/create_utplsql_owner.sql

```diff
@@ -27,9 +27,10 @@ define ut3_tablespace = &3
 
 create user &ut3_user identified by &ut3_password default tablespace &ut3_tablespace quota unlimited on &ut3_tablespace;
 
-grant create session, create sequence, create procedure, create type, create table, create view, create synonym to &ut3_user;
+grant create session, create sequence, create procedure, create type, create table, create view, create synonym, select any dictionary to &ut3_user;
```

Comment:
As this is only used by coverage reporter I dont think we should have it here. There will be lots of DBAs who wount give this grant even its on test server.